### ISSUE TITLE
Missing way to add subnets to transit gateway for AWS 

### ISSUE THREAD

**Issue by ddastoli** [CiscoDevNet/ansible-mso#126]
When creating a new VRF for AWS site, mso_schema_site_vrf_region_hub_network allows to add hub network to the AWS VRF. two features are missing:
1- the possibility to disable the VPN Gateway Router which is automatically attached to the VRF at creation(cisco.mso.mso_schema_site_vrf_region_cidr)
2- there should be also a way to add subnets to the hub network (in which case the subnets should exist already on the VRF through cisco.mso.mso_schema_site_vrf_region_cidr_subnet).

**Comment by lhercot** [comment]
@ddastoli The majority of your request is already supported today as part of mso_schema_site_vrf_region_hub_network and mso_schema_site_vrf_region.

There is a small bug that is fixed by PR #143 which should get merged today and be part of the next release package which should be out early next-week.
The option to assign the subnets to the Hub Network is under the mso_schema_site_vrf_region_cidr_subnet and not the hub_network itself as that is how the API is designed and it also makes more sense for the user to choose it when the subnet is created / updated.

When the new package is available, here is an example on how to achieve this:
```
- name: MSO TESTING
  hosts: localhost
  tasks:
  - name: Set vars
    set_fact: 
      mso_info: &mso_info
        host: {{ mso_ip }}
        username: {{ mso_user }}
        password: {{ mso_password }}
        validate_certs: '{{ mso_validate_certs | default(false) }}'
        use_ssl: '{{ mso_use_ssl | default(true) }}'
        use_proxy: '{{ mso_use_proxy | default(true) }}'
        output_level: '{{ mso_output_level | default("info") }}'

  - name: Ensure tenant ansible_test exist
    cisco.mso.mso_tenant: 
      <<: *mso_info
      tenant: ansible_test
      state: present

  - name: Ensure AWS site is present under tenant ansible_test
    cisco.mso.mso_tenant_site: 
      <<: *mso_info
      tenant: ansible_test
      site: AWS-West
      cloud_account: '000000000000'
      aws_access_key: 1
      secret_key: 0
      state: present

  - name: Ensure schema 1 with Template 1
    cisco.mso.mso_schema_template: 
      <<: *mso_info
      schema: '{{ mso_schema | default("ansible_test") }}'
      tenant: ansible_test
      template: '{{ item }}'
      state: present
    loop:
    - Template 1

  - name: Ensure VRF1 exists
    cisco.mso.mso_schema_template_vrf: 
      <<: *mso_info
      schema: '{{ mso_schema | default("ansible_test") }}'
      template: Template 1
      vrf: VRF1
      state: present

- name: Add a new CIDR in VRF1 at AWS site level
    cisco.mso.mso_schema_site_vrf_region_cidr:
      <<: *mso_info
      schema: '{{ mso_schema | default("ansible_test") }}'
      template: Template 1
      site: AWS-West
      vrf: VRF1
      region: us-west-1
      cidr: 10.0.0.0/16
      primary: true
      state: present

  - name: Add hub network in VRF1 region us-west-1 at AWS site level
    cisco.mso.mso_schema_site_vrf_region_hub_network:
      <<: *mso_info
      schema: '{{ mso_schema | default("ansible_test") }}'
      template: Template 1
      site: AWS-West
      vrf: VRF1
      region: us-west-1
      hub_network:
        name: default
        tenant: infra
      state: present

  - name: Remove VPN Gateway Router at Region for AWS
    cisco.mso.mso_schema_site_vrf_region:
      <<: *mso_info
      schema: '{{ mso_schema | default("ansible_test") }}'
      template: Template 1
      site: AWS-West
      vrf: VRF1
      region: us-west-1
      vpn_gateway_router: false
      state: present

  - name: Add a new subnet to AWS CIDR in VRF1 at site level
    cisco.mso.mso_schema_site_vrf_region_cidr_subnet:
      <<: *mso_info
      schema: '{{ mso_schema | default("ansible_test") }}'
      template: Template 1
      site: AWS-West
      vrf: VRF1
      region: us-west-1
      cidr: 10.0.0.0/16
      subnet: 10.0.0.0/24
      zone: us-west-1a
      hub_network: true
      state: present
```

**Comment by ddastoli** [comment]
Thanks, tested it on cisco-mso-1.0.1.tar.gz and works well !
